Frequently Asked Questions
!!!!!!!!!!!!!!!!!!!!!!!!!!


Alignments for my transcript are not available. What can I do?
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@

The short answer is: not much.

The problem is that NCBI provides alignment data only for current
transcripts against current assemblies.  Historical data are not
available.  The Universal Transcript Archive (UTA) is essentially an
archive of a series of snapshots beginning in 2014, sometimes with
gaps. Any transcript that was obsoleted before UTA started, or existed
only *between* UTA snapshots, will not exist in UTA.

If an alignment for a particular transcript-reference sequence pair
and for a particular alignment method are not available, an exception
like the following will be raised::

  HGVSDataNotAvailableError: No alignments for NM_000018.2 in GRCh37 using splign

Currently, there is no way for users to provide their own alignments.


Why do I get different results on the UCSC browser?
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@

The UCSC Genome Browser uses alignments generated by BLAT, which gives
different results than the official alignments generated by NCBI using
splign.  Although BLAT and splign typically agree, there are many
small differences in ambiguous alignments and even some substantial
differences in a small number of transcripts.  In some cases, the
differences might cause a variant to be interpreted as coding using a
splign alignment and non-coding by a BLAT alignment, or vice
versa. Furthermore, one typically doesn't know which alignment set was
used when publishing a variant. (Yes, that's a hot mess.)


Why do I get different results with Mutalyzer?
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@

Some transcript-genome alignments contain indels.  `hgvs` is careful
to account for these indel discrepancies when projecting variants. In
contrast, Mutalyzer does not account for such discrepancies.
Therefore, the Mutalyzer results will be incorrect when projecting or
validating a variant that is downstream of the first indel.  For
details and other examples, see
https://www.ncbi.nlm.nih.gov/pubmed/30129167.
